src/Clb.hs

```diff
@@ -45,23 +71,25 @@ import Cardano.Ledger.Compactible qualified as L
 import Cardano.Ledger.Core qualified as Core
 import Cardano.Ledger.Keys qualified as L
 import Cardano.Ledger.Mary (MaryValue)
-import Cardano.Ledger.Plutus.TxInfo (txInfoIn', transDataHash')
-import Cardano.Ledger.Pretty.Babbage ()
+import Cardano.Ledger.Plutus.TxInfo (transTxIn, transDataHash)
```

Comment:
This could be `ifdef`-ed to support Atlas (at least in master code version).

Reply:
I am not fully convinced whether cluttering the code with ifdefs is worthwhile. We are not going to support both sets of dependencies as we move forward, and I am completely fine running two branches. If it becomes too heavy burden to carry on, we can go that way, but I would avoid it right now.